templates/semaphore/base.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+def gem_installed?(gem_name)
+  Gem::Specification.find_by_name(gem_name)
+rescue Gem::LoadError
+  nil
+end
+
+file ".semaphore/semaphore.yml", <<~YAML
+  version: v1.0
+  name: Ruby on Rails CI
+  agent:
+    machine:
+      type: e1-standard-2
+      os_image: ubuntu2004
+  auto_cancel:
+    running:
+      when: true
+  fail_fast:
```

Comment:
What do you think about auto stop by time? It can save quota of the semaphore time if you ci stuck.
```
execution_time_limit:
  minutes: 40
```